chart/README.md

```diff
@@ -0,0 +1,94 @@
+# wasmCloud Host Helm Chart
+
+A Helm Chart for installing and scaling wasmCloud hosts inside of Kubernetes.
+
+## Overview
+
+In keeping with our goal of "compatible with, but not dependent on," this Helm chart provides an
+option to users who would like to run wasmCloud hosts inside of pods on Kubernetes. Please note this
+is not the required or recommended way for completely new development, but is meant to be used as a
+way to bridge those already running things in Kubernetes with the new work being done in wasmCloud!
+If you are trying something completely net new, running your hosts in Kubernetes might not be your
+best option.
+
+### Limitations
+
+This chart is purely meant for creating a pool of scalable hosts and not for running your whole
+lattice. Ideally, the main NATS server and the place you'll be controlling your lattice from will
+likely be outside of Kubernetes. If you'd like to run everything inside of Kubernetes, we'd
+recommend deploying your main NATS server with another chart.
+
+## Running the chart
+
+The chart is available in a hosted Helm repository. To add the repository, run the following:
+
+```console
+$ helm repo add wasmcloud https://wasmcloud.github.io/wasmcloud-otp/
```

Comment:
Just leaving a note here as a reminder we'll need to probably update this after turning on GH pages